### PR TITLE
added tolerations for rke controlplane/etcd nodes

### PR DESCRIPTION
**Pull request by paynejacob**
https://github.com/rancher/rancher/issues/29297

### CODE REVIEW

File: packages/rancher-logging/rancher-logging.patch [hunk 1]
```diff
@@ -107,4 +107,10 @@ diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-logging/charts-original/values
 +  cattle:
 +    systemDefaultRegistry: ""
```
Comment (cbron) [inline packages/rancher-logging/rancher-logging.patch:108]: What value was that checkbox setting ?

File: packages/rancher-logging/rancher-logging.patch [hunk 2]
```diff
@@ -107,4 +107,10 @@ diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-logging/charts-original/values
...
 +
-+fluentbit_tolerations: []
++fluentbit_tolerations:
++  - key: node-role.kubernetes.io/controlplane
++    value: "true"
++    effect: NoSchedule
++  - key: node-role.kubernetes.io/etcd
++    value: "true"
++    effect: NoExecute
```
Comment (cbron) [inline packages/rancher-logging/rancher-logging.patch:116]: This is specifically the settings for rke ?

Comment (paynejacob) [reply packages/rancher-logging/rancher-logging.patch:116]: yes, this counters the taints set on rke nodes

**Review by rmweir — APPROVED** [review]
LGTM

File: packages/rancher-logging/rancher-logging.patch (hunk 1 above)
Comment (cbron) [reply packages/rancher-logging/rancher-logging.patch:108]: It was actually setting the fluentbit_tolerations array, so thats fine.

**Review by cbron — APPROVED** [review]

**Pull request merged**